jdaviz/configs/default/plugins/collapse/collapse.py

```diff
@@ -34,6 +34,16 @@ class Collapse(TemplateMixin):
     funcs = List(['Mean', 'Median', 'Min', 'Max', 'Sum']).tag(sync=True)
     selected_func = Unicode('Mean').tag(sync=True)
 
+    # add/replace results for selected_axis != 0
+    add_replace_results = Bool(True).tag(sync=True)
+
+    # selected_viewer for selected_axis == 0
+    # NOTE: this is currently cubeviz-specific so will need to be updated
+    # to be config-specific if using within other viewer configurations.
+    viewer_to_id = {'Left': 'cubeviz-0', 'Center': 'cubeviz-1', 'Right': 'cubeviz-2'}
```

Comment:
Seems weird to have Cubeviz specific setting in a "default" section of plugins but seems to work with the other viz, so probably okay for now.

Reply:
I agree, but couldn't think of a (simple) general way to reference the image viewers by position (suggestions welcomed).  As far as I can tell, collapse is only used in cubeviz, so maybe we should just move the plugin there instead of in default?

Reply:
I don't see why anyone wants to collapse anything besides a cube, so I agree it should be moved but maybe as a separate PR.

Reply:
Honestly I think there are a few things in the package that would make better sense if they were located elsewhere. I agree that there should be a separate PR to do a little reorganizing.